### PR TITLE
Modal component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my modal component

this page contains the modal component of CSS

live link: https://css-companion.netlify.app/components/modal.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of modal components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media

image

### CODE REVIEW

File: component/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+    "pitch pitch"
+   "aside main"
+   "aside main"
+   "aside main"
+   "footer footer"
+}
+
+nav{
+    display: flex;
+    gap: 20px;
```
Comment (sahiba0915) [inline component/modal/modal.css:39]: avoid using pxs unit above 10px

**Review by sahiba0915 — COMMENTED** [review]
everything looks good just remove unwanted spaces

File: component/modal/modal.css (hunk 1 above)
Comment (i-am-a-codegirl) [reply component/modal/modal.css:39]: thanks for lnowlege

File: component/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+    width: 100vw;
+    flex-direction: column;
+    grid-area: nav;
+    
+
```
Comment (utkarshducoder) [inline component/modal/modal.css:44]: waste of space

File: component/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+    box-shadow: 0 0 5px hotpink;
+}
+
+.unorder-li-group {
+    list-style: none;
+    padding: 0.5rem;
+}
+
+
+
```
Comment (utkarshducoder) [inline component/modal/modal.css]: Extra space can increase the time complexity

**Review by utkarshducoder — APPROVED** [review]

File: component/modal/modal.css (hunk 3 above)
Comment (i-am-a-codegirl) [reply component/modal/modal.css]: agree

**Comment by i-am-a-codegirl** [comment]
> everything looks good just remove unwanted spaces

Please approve Sahiba

File: component/modal/modal.css [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+   
+}
+
+main p{
+    letter-spacing: 0.1rem;
+}
+
+aside {
+
+    background: #111827;
```
Comment (khushijohri2001) [inline component/modal/modal.css:108]: use var here for background color

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**